CharlieBackend.Business/Services/Interfaces/IHomeworkStudentService.cs

```diff
@@ -0,0 +1,19 @@
+﻿using CharlieBackend.Core.DTO.HomeworkStudent;
+using CharlieBackend.Core.Models.ResultModel;
+using System.Collections.Generic;
+using System.Security.Claims;
+using System.Threading.Tasks;
+
+namespace CharlieBackend.Business.Services.Interfaces
+{
+    public interface IHomeworkStudentService
```

Comment:
Perhaps we should be adding summaries so there would be less confusion between `HomeworkStudentService` and `HomeworkService`.